doc/src/modules/integrals/integrals.rst

```diff
@@ -46,12 +52,31 @@ Exponential-polynomial functions. These multiplicative combinations of polynomia
 
 even a few nonelementary integrals (in particular, some integrals involving the error function) can be evaluated::
 
-	>>> integrate(exp(-x**2)*erf(x), x)
-	  ____    2
-	\/ pi *erf (x)
-	--------------
-	      4
+	 >>> integrate(exp(-x**2)*erf(x), x)
+   sqrt(pi)*erf(x)**2/4
+
+SymPy's numeric integration (from mpmath) is very accurate and useful and would be preferred
+by many users over using lambdify/scipy. Here is how it is used:
+    >>> from sympy import *
+    >>> x=Symbol('x')
```

Comment:
Put spaces around equals signs.

Reply:
Also this line isn't even needed because x is already defined in previous examples.